spec/requests/companies_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe '/companies', type: :request do
+  describe 'GET /index' do
+    it 'renders a successful response' do
+      get companies_path
+
+      expect(response).to have_http_status(:success)
+    end
+  end
+end
```

Comment:
other tests to add:

* user trying to access new company page but not logged in -> should redirect them to the login page
+ similar tests for other pages that require user to be logged in.

* logged in as User 1 and trying to edit User 1's company -> allowed.
* logged in as User 1 and trying to edit User 2's company -> not allowed. 
+ Similar tests as above for update, destroy actions. 

* correct success responses from all actions